content/en/monitors/create/types/error_tracking.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Error Tracking Monitor
+kind: documentation
+description: Learn about the Error Tracking monitor type.
+further_reading:
+- link: "/real_user_monitoring/error_tracking/"
+  tag: "Documentation"
+  text: "Learn more about Error Tracking"
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "/monitors/notify/downtimes/"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+- link: "/monitors/manage/status/"
+  tag: "Documentation"
+  text: "Check your monitor status"
+---
+
+## Overview
+
+Once [Real User Monitoring][1] or [APM][2] is enabled for your organization, you can create an [Error Tracking][3] monitor to alert you when an issue starts, when it has a high impact, and when it starts regressing.
+
+## Create an Error Tracking monitor
+
+To create an Error Tracking monitor in Datadog, navigate to [**Monitors** > **New Monitor** > **Error Tracking**][4].
+
+<div class="alert alert-info"><strong>Note</strong>: There is a default limit of 1,000 Error Tracking monitors per account. <a href="/help/">Contact Support</a> to lift this limit for your account.</div>
+
+### Select the alerting condition
+
+Alerts are grouped by issue ID and are based on error volume.
+
+Choose **Count** to alert on issues with a high number of errors and **New Issue** to alert on issues that occur for the first time.
+
+### Define the search query
+
+{{< tabs >}}
+{{% tab "Count" %}}
+
+For **Web and Mobile Apps**, select **Count**. As you expand your search filter, the Count graph above **Select the alerting condition** updates.
+
+1. Optionally, select a query suggestion such as **Error Occurrences**, **Impacted Users**, and **Impacted Sessions** to alert on. 
```

Comment:
The information here is repeated in lines 50-54, where more information is given. I would just delete this, since you tell them to do it later on